src/table/utils/handle-scroll.js

```diff
@@ -13,11 +13,11 @@ export const handleHorizontalScroll = (evt, isRTL, memoedContainer) => {
     // evt.deltaX increasingly negative as you scroll towards left,
     // increasingly positive as you scroll towards right
     const scrollRight = maxScrollableWidth + ScrollLeftWidth;
-    if (maxScrollableWidth > 0 && (scrollRight <= 0 || scrollRight > maxScrollableWidth)) {
+    if (maxScrollableWidth > 0 && (scrollRight < 0 || scrollRight > maxScrollableWidth + 1)) {
```

Comment:
codeclimate does have a point here. The only difference is using `scrollRight` instead of `ScrollLeftWidth`, and `min` instead of `max`. Could refactor this if it doesn't hurt the readability. You could also early return when `maxScrollableWidth > 0` is false